### PR TITLE
 use var where apparent 

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by JimBobSquarePants** [comment]
Got a failing test on here.
https://ci.appveyor.com/project/six-labors/imagesharp/builds/26661327/job/sv4nakp1mge7cl3k/tests

https://travis-ci.org/SixLabors/ImageSharp/builds/571259943?utm_source=github_status&utm_medium=notification

File: tests/ImageSharp.Tests/Colorspaces/RgbTests.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ public void RgbAndRgb24Operators()
             const byte g = 128;
             const byte b = 255;
 
-            Rgb24 rgb24 = new Rgb(r / 255F, g / 255F, b / 255F);
+            var rgb24 = new Rgb(r / 255F, g / 255F, b / 255F);
```
Comment (JimBobSquarePants) [inline tests/ImageSharp.Tests/Colorspaces/RgbTests.cs]: Bug is in the test itself. We're not implicitly casting anymore.

**Comment by SimonCropp** [comment]
@JimBobSquarePants notes. will have a look

File: tests/ImageSharp.Tests/Colorspaces/RgbTests.cs (hunk 1 above)
Comment (SimonCropp) [reply tests/ImageSharp.Tests/Colorspaces/RgbTests.cs]: will revert that one

**Comment by SimonCropp** [comment]
now the weird thing is, now when i build i get a heap of errors about "dont use var".

so it think there might be something wrong with the editor config

@JimBobSquarePants can u clone and verify?

**Comment by JimBobSquarePants** [comment]
@SimonCropp Will do!

**Comment by JimBobSquarePants** [comment]
@SimonCropp Not seeing any warnings about that at all. Neither does Appveyor. 

https://ci.appveyor.com/project/six-labors/imagesharp/builds/26661327/job/sv4nakp1mge7cl3k#L486

**Comment by SimonCropp** [comment]
@JimBobSquarePants ok i deleted and re-clones. no more errors. i think is must have been my beta of r# (which i moved to stable a few days ago) had messed with my intelisense via some old cached files.

**Comment by JimBobSquarePants** [comment]
@SimonCropp that’s great to hear! I’m amazed though you’re able to use R# with this project, it grinds my system to a halt!

**Pull request merged**